### PR TITLE
[My Registrations][ENG-2627][ENG-2586] Add View button and tags

### PR DESCRIPTION
**Pull request by adlius**
- Ticket: [ENG-2627][ENG-2586]
- Feature flag: n/a

## Purpose

Modify the `NodeCard` component to show labels for fields and add a "View" button.

## Summary of Changes

Screenshot from 2021-04-20 04-42-18

## QA Notes

[ENG-2627]: https://openscience.atlassian.net/browse/ENG-2627
[ENG-2586]: https://openscience.atlassian.net/browse/ENG-2586

### CODE REVIEW

File: translations/en-us.yml [hunk 1]
```diff
@@ -598,7 +598,11 @@ node_card:
             withdrawn: Withdrawn
             archiving: Archiving
     tags: 'Tags:'
+    schema: 'Schema:'
```
Comment (futa-ikeda) [inline translations/en-us.yml:601]: It doesn't look like we use "Schema" for user-facing language, but instead call it "Registration Type" so I'm checking with Mark if "Schema" is understandable, or if we should call it something else

Comment (futa-ikeda) [reply translations/en-us.yml:601]: Looks like the language we want to use going forward is "Registration Template"

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
Minor language change, but otherwise looks good!

File: translations/en-us.yml (hunk 1 above)
Comment (fabmiz) [reply translations/en-us.yml:601]: +1

File: lib/osf-components/addon/components/node-card/styles.scss [hunk 2]
```diff
@@ -74,7 +74,11 @@
...
 
-    strong {
-        font-weight: 600;
+    div {
+        display: inline-block;
     }
 }
+
+.Labels {
+    font-weight: 600;
```
Comment (fabmiz) [inline lib/osf-components/addon/components/node-card/styles.scss:83]: suggestion
```suggestion
    font-weight: 600;
    &::after {
        content: ": ";
    }
```

File: lib/osf-components/addon/components/node-card/template.hbs [hunk 3]
```diff
@@ -100,32 +100,79 @@
...
+                        data-test-schema-label
+                        local-class='Labels'
+                    >
+                        {{t 'node_card.schema'}}
+                    </strong>
                     {{@node.registrationSchema.name}}
                 </div>
             {{/if}}
-            <div local-class='NodeCard__authors'>
+            <div data-test-contributors
```
Comment (fabmiz) [inline lib/osf-components/addon/components/node-card/template.hbs:123]: [HTML dl](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) is more suitable for `key-value` type of data. Might also be best for accessibility. It indicates that this is a metadata list for the registration.

**Review by fabmiz — DISMISSED** [review]
Looks good. Mostly suggestions for better HTML. Request change because the use of `schema` in user-facing language as @futa-ikeda mentioned.

File: translations/en-us.yml [hunk 4]
```diff
@@ -598,7 +598,11 @@ node_card:
...
+    contributors: 'Contributors:'
+    description: 'Description:'
```
Comment (fabmiz) [inline translations/en-us.yml:603]: With the pseudo element `::after{ content: ": "; };` on `.Labels` no need for ":" here.

**Comment by brianjgeiger** [comment]
We should probably make the button styling match the [Draft Node card's styling](https://github.com/CenterForOpenScience/ember-osf-web/pull/1174). I made a comment on the Draft Node card PR that we should also match the field label styling. Basically, the styling between these cards should probably be a bit closer to each other for consistency. Should we check with Ash and Mark to see which they prefer for what and consolidate?

**Comment by brianjgeiger** [comment]
Also, looks like [a test has failed](https://github.com/CenterForOpenScience/ember-osf-web/pull/1173/checks?check_run_id=2353027328) after merging in develop (which shouldn't have caused a test failure, so may be probabilistic?).

File: lib/osf-components/addon/components/node-card/styles.scss [hunk 5]
```diff
@@ -29,6 +29,15 @@
     }
 }
 
+.NodeCard__body dt,
+dd {
+    display: inline-block;
+}
+
+.NodeCard__body dl {
+    margin-bottom: 0;
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/node-card/styles.scss:38]: From UX Consults: let's bump this up to 10px

**Review by brianjgeiger — DISMISSED** [review]

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
Just a couple of leftover classes that I see

File: lib/osf-components/addon/components/node-card/template.hbs [hunk 6]
```diff
@@ -98,34 +99,88 @@
...
                     </div>
                 {{/if}}
-                <div local-class='NodeCard__schema-name'>
-                    {{@node.registrationSchema.name}}
+                <div data-test-contributors
+                    local-class='NodeCard__authors'
+                >
+                    <dt
+                        data-test-contributors-label
+                        local-class='Labels'
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/node-card/template.hbs:129]: Sorry, this class is still here

File: lib/osf-components/addon/components/node-card/template.hbs [hunk 7]
```diff
@@ -98,34 +99,88 @@
...
-                        @taggable={{@node}}
-                        @inline={{true}}
-                    />
+                <div
+                    data-test-description
+                    local-class='NodeCard__description'
+                >
+                    <dt
+                        data-test-description-label
+                        local-class='Labels'
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/node-card/template.hbs:146]: and here

File: lib/osf-components/addon/components/node-card/template.hbs [hunk 8]
```diff
@@ -98,34 +99,88 @@
...
                 </div>
-            {{/if}}
+                {{#if (and this.showTags @node.tags)}}
+                    <div
+                        data-test-tags
+                        local-class='NodeCard__tags'
+                    >
+                        <dt
+                            data-test-tags-label
+                            local-class='Labels'
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/node-card/template.hbs:161]: and here

**Review by futa-ikeda — CHANGES_REQUESTED** [review]

**Review by futa-ikeda — APPROVED** [review]
🚀

**Review by brianjgeiger — APPROVED** [review]

**Review by futa-ikeda — APPROVED** [review]

**Pull request merged**